internal/security/proxy/requestor.go

```diff
@@ -19,11 +19,13 @@ import (
 	"crypto/tls"
 	"net/http"
 	"time"
-
-	"github.com/edgexfoundry/edgex-go/internal"
 )
 
-func NewRequestor(skipVerify bool, timeout int) internal.HttpCaller {
+type Requestor interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+func NewRequestor(skipVerify bool, timeout int) Requestor {
```

Comment:
`timeoutInSecond` (instead of just `timeout`) seems better to read at first glance of this func before looking into the details of code below to figure it out.

Reply:
agree and will change the name of the parameter.